src/components/Widget/GroupsWidget/GroupsWidget.js

```diff
@@ -72,3 +54,26 @@ export default class GroupsWidget extends Component {
     )
   }
 }
+
+export function GroupCard ({ group, routeParams, className }) {
```

Comment:
Ehh, Aaron said this version of the groupcard will probably be overwritten once we settle on the new version from the Group Explorer :(

Reply:
Does this version now show up in the context of the map?

Reply:
Yeah thats what I decided to use in the map drawer for now (at Aaron’s suggestion)